collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/SwitchProjectTest.kt

```diff
@@ -103,5 +103,10 @@ class SwitchProjectTest {
             // Check instances
             .clickSendFinalizedForm(1)
             .assertText("Two Question")
+            .pressBack(MainMenuPage())
+
+            // Check entities
+            .openEntityBrowser()
+            .assertTextDoesNotExist("people")
```

Comment:
You check if it doesn't exist for the demo project but you never check if it existed for the second one.

Reply:
Yeah good point. We don't need to test that the entity is created in the current project as that's covered by `EntityFormTest`, but it does feel a little strange to make one assertion and not the other. I'll add it.